base.scss

```diff
@@ -0,0 +1,68 @@
+h1 {
+    font-family: Lucida Sans Unicode;
+    border-bottom: 1px solid silver;
+}
+
+#page-wrapper {
+    width: 500px;
+    position: absolute;
```

Comment:
With fixed width aligning container in the middle using margin: 0 auto would be more effective